.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,20 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: "[Bug] "
+labels: bug
+assignees: ''
+
+---
+
+## Bug report:
+* **Expected Behavior**: 
+* **Actual Behavior**: 
```

Comment:
feel free to ignore the comment, but I prefer to have these 2 before `Steps to Reproduce` so it's more like a story: after doing this I should see this, but I see this

Reply:
I think this way is the standard and I like to know the problem before the steps to reproduce actually 😅. But same here, feel free to ignore

Reply:
Yes, not sure if there is a standard I saw all kind of orders and I think each one has his pro/cons. I will merge/make prs using this and if is not working for a specific project it can be updated.